docs/releases/3.1.txt

```diff
@@ -672,6 +672,10 @@ Miscellaneous
 
 * The undocumented ``django.contrib.postgres.fields.jsonb.JsonAdapter`` class
   is removed.
+* The ``URLify`` JavaScript function that is used by
```

Comment:
Add to `3.2.txt`, 3.1 has already forked off master

Also one blank line between each entry

Reply:
Oops, thanks! Fixed by amending the commit and force-pushed.